docs/configuration.rst

```diff
@@ -5,8 +5,33 @@ Configuration
    :caption: application.cfg
 
    # Sample application.cfg file
+   import os
+
+   # Database paths
+   CELERY_BROKER_URL = 'redis://redis'
```

Comment:
For development installation, we don't need to set this, because the developer will be running redis on localhost.